libs/sdk-ui-dashboard/src/model/commandHandlers/dashboard/common/stateInitializers.ts

```diff
@@ -28,6 +30,10 @@ import { DashboardContext, PrivateDashboardContext } from "../../../types/common
 import { ObjRefMap } from "../../../../_staging/metadata/objRefMap";
 import { ExtendedDashboardWidget } from "../../../types/layoutTypes";
 import { getPrivateContext } from "../../../store/_infra/contexts";
+import { loadAvailableDisplayFormRefs } from "./loadAvailableDisplayFormRefs";
+import { PromiseFnReturnType } from "../../../types/sagas";
+import { update } from "lodash/fp";
```

Comment:
Please use 
```
import update from "lodash/fp/update"
```
instead, it three-shakes better. We need to imporve our ESLint/dep-cruiser setup to catch this :)